### PR TITLE
Changed bottom images on index.html

### PR DESCRIPTION
**Pull request by rayning0**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -78,39 +78,33 @@
       </div>
     </div>
   </div>
-
+Pop    Alternative    Hip-Hop    Dance Rock    Country
```
Comment (jacobwgillespie) [inline index.html]: This line should be removed

**Comment by jacobwgillespie** [comment]
The images need to be converted to JPEG as the PNGs are too large for the web.  Ideally after being converted to JPEGs they are also run through an optimizer (like https://imageoptim.com/) too.

**Comment by rayning0** [comment]
Done. Changed PNG back to JPG images. Aline had also asked me to change order of genres on index.html, to (left to right): pop, alternative, hip-hop, dance, rock, country.

**Comment by jacobwgillespie** [comment]
Retroactive LGTM :smile:

**Pull request merged**